src/test/java/uk/gov/justice/hmpps/prison/api/resource/HealthIntTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.hmpps.prison.api.resource;
+
+import org.junit.Test;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.RequestEntity;
+import uk.gov.justice.hmpps.prison.api.resource.impl.ResourceTest;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.OK;
+
+public class HealthIntTest extends ResourceTest {
+
+    @Test
+    public void healthReportsOk() {
+        getAndVerifyStatusUp("/health");
+    }
+
+    @Test
+    public void dbReportsOk() {
+        getAndVerifyStatusUp("/health/db");
+    }
+
+    @Test
+    public void pingReportsOk() {
+        getAndVerifyStatusUp("/health/ping");
+    }
+
+    @Test
+    public void livenessReportsOk() {
+        getAndVerifyStatusUp("/health/liveness");
+    }
+
+    @Test
+    public void readinessReportsOk() {
+        getAndVerifyStatusUp("/health/readiness");
+    }
+
+    private void getAndVerifyStatusUp(final String url) {
+        final var response = testRestTemplate.exchange(
```

Comment:
so prison-api is still using rest template, who knew??

Reply:
Yeah a pain.  Wanted to use webclient...